packages/sdk/src/clients/optimisticOracle/README.md

```diff
@@ -0,0 +1,27 @@
+# UMA SDK Optimistic Oracle Client
```

Comment:
thanks for adding the docs!

Reply:
one question: we have all these really useful README files through the repo (mainly in the `sdk` package). I feel that people don't really see them as they are kind of hidden away, nested within a bunch of directories. do you have any ideas on a better way to export these? should these SDK docs be within https://docs.umaproject.org/ to show people how to use the SDK?

Reply:
i've tried to create a links from the readme in the root of the repo to each readmes in each major folder. in general though i dont think the sdk is quite ready for external use, so im not too concerned about making the docs visible.  i think right now the parts of the sdk which are most useful externally is the across client, which is up on the doc site, and potentially the optimistic oracle client, since both of these have an application.  idk though just my rough thoughts on this